File: package/main/src/Math/radToDeg.ts
import { division } from "./division";

/**
 * Converts radians to degrees
 * @param  {number} x Angle in radians
 * @returns {number} Angle in degrees
 * @example radToDeg(Math.PI); // 180
 * @description
 * Uses the formula: degrees = radians * (180/π)
 */
export const radToDeg = (x: number) => division(x, division(Math.PI, 180));

File: package/umt_python/src/math/rad_to_deg.py
import math

from .division import division


def rad_to_deg(x: float) -> float:
    """
    Converts radians to degrees.

    Args:
        x: Angle in radians.

    Returns:
        Angle in degrees.

    Example:
        >>> rad_to_deg(math.pi)
        180.0
    """
    return division(x, division(math.pi, 180))
